lib/referee-sinon.test.js

```diff
@@ -628,6 +628,49 @@ describe("referee-sinon", function() {
             });
         });
 
+        describe("calledOnceWithExactly", function() {
+            it(
+                "fails when not called with spy",
+                requiresSpy("calledOnceWithExactly")
+            );
+
+            it("fails when spy is explicitly passed null", function() {
+                var spy = sinon.spy();
+                spy(null, "Hey");
+
+                refute.calledOnceWithExactly(spy, null, "Hey!");
+            });
+
+            context(
```

Comment:
Why a `context` for a single `it`?

Reply:
I find the pattern of using context in the `it` statements taxing.

```
it("should something when some condition
```

vs

```
context("when some condition"
```

To me, the context should always be in `describe` or `context` blocks.

Reply:
I see. I've personally moved away from "should ..." entirely and use a verb to start with. In this case I would do `it("fails when called more than once with same arguments",`.